docs/content/en/concepts/quickstart-tenancy-and-apis.md

```diff
@@ -309,19 +369,14 @@ some way of securely distinguishing them.
 Each `APIExport` is allocated a randomized private secret - this is currently just a large random number - and a public
 identity - just a SHA256 hash of the private secret - which securely identifies this `APIExport` from others.
 
-This is important because an `APIExport` makes a virtual workspace available to interact with all instances of a
-particular `APIResourceShema`, and we want to make sure that users are clear on which service provider `APIExports` they
-are trusting and only the owners of those `APIExport` have access to their resources via virtual workspaces.
+This is important because an `APIExport` makes service endpoints available to interact with all instances of a particular `APIResourceShema`, and we want to make sure that users are clear on which service provider `APIExports` they are trusting and only the owners of those `APIExport` have access to their resources via the service endpoints.
 
-Q: Why do you have to use `--all-namespaces` with the apiexport virtual workspace?
+Q: Why do you have to use `--all-namespaces` with the `APIExport` service endpoint?
 
-A: Think of this virtual workspace as representing a wildcard listing across all workspaces. It doesn't make sense to
-look at a specific namespace across all workspaces, so you have to list across all namespaces too.
+A: Think of this endpoint as representing a wildcard listing across all workspaces. It doesn't make sense to look at a specific namespace across all workspaces, so you have to list across all namespaces too.
 
-Q: If I attempt to use an `APIExport` virtual workspace before there are any `APIBindings` I get the "Error from server
-(NotFound): Unable to list ...: the server could not find the requested resource". Is this a bug?
+Q: If I attempt to use an `APIExport` endpoint before there are any `APIBindings` I get the "Error from server (NotFound): Unable to list ...: the server could not find the requested resource". Is this a bug?
 
 A: It is a bug. See <https://github.com/kcp-dev/kcp/issues/1183>
```

Comment:
This issue was closed, is the above still valid?

Reply:
Yes it is. The resolution was only partial, i.e. not working with shards, and not ported to the new API. The preferred approach for now would be to amend the virtual workspace so that an empty list is returned:
https://kubernetes.slack.com/archives/C021U8WSAFK/p1674303045697859?thread_ts=1674243613.104869&cid=C021U8WSAFK